comprobo_final_project/scripts/april_tags.py

```diff
@@ -0,0 +1,53 @@
+#!usr/bin/env python
+
+"""
+Got Apriltags absolute position
+"""
+
+import rospy
+from geometry_msgs.msg import Twist, Point, PoseStamped
+import tf
+
+class April(object):
+    """
+    Basically for now it just takes data from the Apriltags and can send it somewhere else
+    """
+
+    def __init__(self):
+        rospy.init_node("april")
+
+        rospy.Subscriber("/STAR_pose_continuous", PoseStamped, self.getPos, queue_size=10)
+
+        self.position = None
+        self.orient = None
+
+    def getPos(self, data):
+        """
+        just takes the position and does something with it
+        """
+
+        self.position = data.pose.position.x, data.pose.position.y, data.pose.position.z
```

Comment:
pose.position will give you an object that you then additionally have to call .x .y and .z on to get the values. I thought that just parsing that out such that you can just directly access the numbers themselves would be easier.